README.md

```diff
@@ -106,6 +106,43 @@ bundle exec rake
 
 To run the feature specs you will need Chrome installed.
 
+### Running a live console
+
+**Accessing a live console is very risky and should only be done as a last
+resort. This should only be done in pairs, and mutating any live data is
+STRONGLY discouraged.**
+
+The console will be ran inside a container instance and won't be on one of the
+web servers, however it will have access to the database.
+
+#### Through the Azure UI
+
+- Navigate to the container instance resource (eg. `s118d01-app-worker-aci`)
+- Then go to 'Containers' under 'Settings'
+- With the container selected go to the 'Connect' tab
+- Choose the start up command (`/bin/bash` is recommended) and connect
+
+#### Through the Azure CLI
+
+We have a helpful script you can run that will connect you to the right
```

Comment:
It would be worth mentioning that they need to have the Azure CLI installed.

Reply:
Done